Extensions/Ansible/Src/Tasks/Ansible/ansibleUtils.ts

```diff
@@ -195,6 +197,10 @@ function beginRequestInternal(request: WebRequest): Promise<WebResponse> {
     });
 }
 
+export function getUniqueInventoryFilePath(): string {

```

Comment:
consider renaming to getTemporaryInventoryFilePath

Reply:
Done.